modules/retail/pages/retail-uyuni-server-setup.adoc

```diff
@@ -136,42 +136,79 @@ spacewalk-repo-sync -c uyuni-proxy-stable-leap-152-x86_64
 
 For more information about creating activation keys, see xref:client-configuration:activation-keys.adoc[].
 
-Activation key for proxy must contain these child channels:
+The activation key for a proxy (branch server) must contain these child channels:
 
 * openSUSE Leap 15.2 Updates (x86_64)
 * Uyuni Client Tools for openSUSE Leap 15.2 (x86_64)
 * Uyuni Proxy Stable for openSUSE Leap 15.2 (x86_64)
 
-Activation key for retail terminal based on Leap 15.2 must contain these child channels:
+The activation key for retail terminals based on openSUSE Leap 15.2 must contain these child channels:
 
 * openSUSE Leap 15.2 Updates (x86_64)
 * Uyuni Client Tools for openSUSE Leap 15.2 (x86_64)
 
-Then run:
 
-mgr-create-boostrap-repo (but it shouldn't be necessary as it should be created automatically)
 
-Bootstrap proxy via web UI or script using activation key created for proxy.
 
-Uyuni Proxy specific steps
+== Bootstrapping
 
-NOTE: This proxy is going to be used as a branch server. (There is no place where Uyuni uses term Branch
-Server.)
+Bootstrap repositories are now created automatically.
+If something is missing, run the [command]``mgr-create-boostrap-repo`` manually.
 
-Check "Uyuni Proxy Stable for openSUSE Leap 15.2 (x86_64)" channel is assigned to the proxy at its system
-profile page.
-Then run:
 
+
+=== Proxy
+
+For more information about proxies, see xref:installation:uyuni-proxy-registration.adoc[].
+
+Bootstrap the proxy with the {webui} or on the command line using activation key created for proxy.
+
+
+[NOTE]
+====
+This proxy is going to be used as a retail branch server.
+====
```

Comment:
Probably better to take this out of an admon, and add it to the exposition at the top of the section :point_up:

Reply:
ok, done